docs/reference/rest_api_communities.md

```diff
@@ -9,6 +9,30 @@
 | Name       | Type   | Location | Description                                                  |
 | ---------- | ------ | -------- | ------------------------------------------------------------ |
 | `accept`   | string | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json` |
+| `access`   | object | body     | [Access](#community-access) of the community. |
+| `id`       | string | body     | Required, url-compatible, max 100 char. The id of the community that will be used in the community's URL. |                                                  |
+| `metadata` | object | body     | [Metadata](#community-metadata) of the community. |
+
+### Community access
+
+| Name              | Type   | Location | Description                                                   |
+| ----------        | ------ | -------- | ------------------------------------------------------------- |
+| `visibility`      | string | body     | Required, one of `"public"` or `"restricted"`. Visible by the public or restricted to those who have access. |
+| `member_policy`   | string | body     | Required, one of `"open"` or `"closed"`. Can people ask to be part of the community (open) or not (closed)? |
+| `record_policy`   | string | body     | Required, one of `"open"` or `"closed"` or `"restricted"`. Can records be submitted to the community (open) or not (closed)? |
```

Comment:
What does `"restricted"` mean? 

The `public`/`restricted` and `open`/`closed`/`restricted` have been a bit confusing on our side. What about `public`/`private`?

Reply:
Restricted will be removed (see https://github.com/inveniosoftware/invenio-communities/issues/499) and hence will be either ``open`` (everyone) or ``closed`` (members only)